tests/common/platform/interface_utils.py

```diff
@@ -91,6 +92,11 @@ def check_interface_status(dut, asic_index, interfaces, xcvr_skip_list):
     @param dut: The AnsibleHost object of DUT. For interacting with DUT.
     @param interfaces: List of interfaces that need to be checked.
     """
+    # If no interfaces to check, return True
```

Comment:
If there are no interfaces passed to this function, we should return false- I would expect this function to return true only if there are interfaces, and they are up